doc/en/user/source/extensions/wps-download/rawDownload.rst

```diff
@@ -394,6 +394,32 @@ This time, when issued (and process has finished on the server), the GET request
   </wps:ExecuteResponse>
 
 
+Raster Output Format and Response mime-type
++++++++++++++++++++++++++++++++++++++++++++
+By default, the Downloaded raster get finally zipped (with the zip also containing the associated sld). 
+In some cases, this can be unneeded, especially if the output TIFF already has some type of internal compression or if we simply want to get back the TIFF output file without the ancillary sld. Think about downloading an RGB TIFF: the default raster.sld style won't add anything useful to the output. In that case you can specify image/tiff in the Response's output mimeType so that the output TIFF will be provided as is.
```

Comment:
```suggestion
In some cases, this can be unnecessary, especially if the output TIFF already has some type of internal compression or if we simply want to get back the TIFF output file without the ancillary SLD. Let's consider downloading a RGB TIFF: the default raster.sld style won't add anything useful to the output. In that case it's possible to specify ``image/tiff ``in the Response's output ``mimeType``: the output TIFF will be provided as is, without extra steps of compression and file management.
```